QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -16,8 +16,20 @@
 	private static FloatOutput rightBackMotor = Igneous.makeTalonMotor(3, Igneous.MOTOR_FORWARD, .1f);
 	private static FloatOutput rightMotors = FloatMixing.combine(rightFrontMotor, rightBackMotor);
 	private static FloatOutput leftMotors = FloatMixing.combine(leftFrontMotor, leftBackMotor);
+	public static FloatOutput allMotors = FloatMixing.combine(leftMotors, rightMotors);
+	public static FloatOutput rotate = new FloatOutput() {
+		public void set(float value) {
+			leftMotors.set(value);
+			rightMotors.set(-value);
+		}
+	};
```

Comment:
Any particular reason that you're doing this instead of using `combine` and `negate`?